core/trino-main/src/main/java/io/trino/operator/OperatorContext.java

```diff
@@ -697,6 +697,18 @@ public ListenableFuture<Void> setBytes(long bytes)
             return blocked;
         }
 
+        @Override
+        public ListenableFuture<Void> addBytes(long delta)
```

Comment:
you can use this method in `setBytes` to avoid duplication, same for other classes

Reply:
It's deduplicated in other cases. For this one I would need to introduce more synchronization and I'd like to avoid that.